core/workspace_svg.js

```diff
@@ -300,8 +300,18 @@ Blockly.WorkspaceSvg.prototype.addFlyout_ = function() {
 };
 
 /**
- * Resize this workspace and its containing objects.
+ * Resize the parts of the workspace that change when the workspace
+ * contents (e.g. block positions) change.
  */
```

Comment:
Add a note that calling scrollbar.resize() will also scroll the workspace contents if needed.